starfish/core/_display.py

```diff
@@ -226,7 +226,15 @@ def display(
         from qtpy.QtWidgets import QApplication
         app = QApplication.instance() or QApplication([])
 
-        viewer = Viewer()
+        viewer = Viewer(
+            axis_labels=[
+                Axes.ROUND.value,
+                Axes.CH.value,
+                Axes.ZPLANE.value,
```

Comment:
I would grab these three from starfish/core/imagestack/dataorder.py

Reply:
Will do!

Reply:
@ttung , it looks like `AXES_DATA` contains names and orders for round, channel, and z, but not x and y. Should I just assume (y, 3) and (x, 4)? Or is the axis ordering for y and x stored elsewhere?

Reply:
(you can rotate views in `napari`, so the y and x axes may get displayed)

Reply:
Unfortunately, Y and X are not stored anywhere.  It has always been assumed Y then X.

Reply:
Sounds good!

Reply:
@ttung, I changed it such that the axis order comes from `starfish.core.imagestack.dataorder`.